src/app.js

```diff
@@ -2,0 +1,28 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const fun = (firsts, seconds) => {
+  if (!fs.existsSync(firsts)) {
+    console.error(`Source file does not exist: ${firsts}`);
+
+    return;
+  }
+
+  const destination =
+    fs.existsSync(seconds) && fs.statSync(seconds).isDirectory()
+      ? path.join(seconds, path.basename(firsts))
+      : seconds;
```

Comment:
This logic correctly determines if the destination is a directory and constructs the destination path accordingly. However, it might be useful to check if the destination directory is writable before attempting to move the file.